pyproject.toml

```diff
@@ -17,27 +17,28 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = "^3.9"
+aiocontextvars = "^0.2.2"
 aiohttp = "^3.10.3"
+aiohttp-tokenauth = "^0.0.2"
```

Comment:
I think we may not want to install this lib with last modification 5 years ago?
I would rather copy the logic in hydra directly since the usage looks clean and the code simple enough?
What do you think?

Reply:
That's a very good idea, it will greatly simplifies the code, make it more readable and improve maintanability.
Just commited changes to do that.